src/test/java/com/google/api/codegen/testdata/csharp_gapic_client_library.baseline

```diff
@@ -2691,8 +2723,8 @@ namespace Google.Example.Library.V1
             CallSettings callSettings = null) => AddCommentsAsync(
                 new AddCommentsRequest
                 {
-                    Name = name,
-                    Comments = { comments },
+                    Name = GaxPreconditions.CheckNotNullOrEmpty(name, nameof(name)),
+                    Comments = { GaxPreconditions.CheckNotNull(comments, nameof(comments)) },
```

Comment:
Yes, "comments" is required here.
If it's required, I think I'm happy that we forbid the user to pass null. I was also wondering about enforcing that it's non-empty, but this possibly seems like a step too far...